app/views/fragments/amiUsage.scala.html

```diff
@@ -0,0 +1,62 @@
+@(instances: List[Instance], launchConfigs: List[LaunchConfiguration], emptyChar: String = "-")
+
+@if(instances.nonEmpty || launchConfigs.nonEmpty) {
+    <div class="ami-usage-title">
+    <span class="ami-usage-title--label">AMI Usage: </span>
+        <span class="ami-usage-title--info">@instances.length instances, @launchConfigs.length launch configurations</span>
+    </div>
+} else {
+    <div class="ami-usage--message">No instances or launch configurations are using this AMI</div>
+}
+@if(instances.nonEmpty) {
+    <div class="ami-usage--message">Instances</div>
+    <table class="striped ami-usage--table">
+        <thead>
+            <tr>
+                <th class="instance-details-table__heading">Stack</th>
+                <th class="instance-details-table__heading">Stage</th>
+                <th class="instance-details-table__heading">App</th>
+                <th class="instance-details-table__heading">State</th>
+                <th class="instance-details-table__heading">Instance name</th>
+                <th class="instance-details-table__heading">Ip</th>
+                <th class="instance-details-table__heading">Created at</th>
+            </tr>
+        </thead>
+        <tbody>
+        @for(instance <- instances) {
+            <tr>
+                <td class="instance-details-table__entry">@instance.stack.getOrElse(emptyChar)</td>
+                <td class="instance-details-table__entry">@instance.stage.getOrElse(emptyChar)</td>
+                <td class="instance-details-table__entry">@instance.app</td>
+                <td class="instance-details-table__entry">@instance.vendorState</td>
+                <td class="instance-details-table__entry">@instance.instanceName</td>
+                <td class="instance-details-table__entry">@instance.ip</td>
+                <td class="instance-details-table__entry">@instance.createdAt</td>
+            </tr>
+        }
+        </tbody>
+    </table>
+}
+@if(launchConfigs.nonEmpty) {
+    <div class="ami-usage--message">Launch Configurations</div>
+    <table class="striped ami-usage--table">
+        <thead>
+            <tr>
+                <th class="instance-details-table__heading">Account</th>
+                <th class="instance-details-table__heading">Name</th>
+                <th class="instance-details-table__heading">Created at</th>
+                <th class="instance-details-table__heading">Instance type</th>
+            </tr>
+        </thead>
+        <tbody>
+        @for(launchConfig <- launchConfigs) {
+            <tr>
+                <td class="instance-details-table__entry">@launchConfig.meta.origin.accountName</td>
+                <td class="instance-details-table__entry">@launchConfig.name</td>
+                <td class="instance-details-table__entry">@launchConfig.createdTime</td>
+                <td class="instance-details-table__entry">@launchConfig.instanceType</td>
+            </tr>
+        }
+        </tbody>
+    </table>
+}
```

Comment:
This is a thought rather than a request: Could launch configs go above instances? It should always be a shorter list so won't become buried at the bottom if first. Would that make more sense??